helix-core/src/test/java/org/apache/helix/task/TestGetLastScheduledTaskExecInfo.java

```diff
@@ -75,6 +75,10 @@ public void testGetLastScheduledTaskExecInfo() throws Exception {
 
     // Stop and delete the queue
     _driver.stop(queueName);
+    TestHelper.verify(() -> {
```

Comment:
Test bug fix: if the job queue is deleted right after stopping, it is possible that current states are not dropped on the instances. This is because the RUNNING to STOPPED message may still be pending when STOPPED to DROPPED message is generated, meaning the latter message is not sent; however, during the next pipeline cycle the job queue may have been deleted already, so the STOPPED to DROPPED message is never generated again. Adding this check to make the test non-flaky.